modin/pandas/general.py

```diff
@@ -759,3 +759,11 @@ def get_names(obj):
         return list(names[0]) if is_list_like(names[0]) else [names[0]]
     else:
         return None
+
+
+def to_timedelta(arg, unit=None, errors="raise"):
+    if isinstance(arg, Series):
+        arg = arg._to_pandas()
```

Comment:
I think we can implement this case, where the arg is a `Series`, as a `map` without materializing the `arg`. Something like `s._query_compiler._modin_frame.map(lambda s: pandas.to_timedelta(s.squeeze(1)).to_frame()).to_pandas()` seem to work. Note the `squeeze` within the partition function because `pandas.to_timedelta` can take a series but can't take a dataframe, and the `to_frame` because (I think) the partitions have to have dataframes internally and can't have series. You may have to add a new query compiler function for this.

Reply:
@mvashishtha that's right, the partitions need to have dataframes internally. We don't treat Series separately. We just treat Series a special case of DataFrames, so we may need to have another query compiler function for this.